src/assets/scss/components/15-years/15-years.scss

```diff
@@ -59,25 +71,30 @@ $font-family-monospace: 'Roboto Mono', monospace;
 }
 
 @mixin body-text() {
-	font-size: rem-calc(14);
+	font-size: calc(18 / 320 * 100vw);
 	letter-spacing: -0.02em;
 	line-height: 1.35;
 
-	@include breakpoint('large') {
-		font-size: rem-calc(16);
+	// @include breakpoint('large') {
+	// 	font-size: rem-calc(16);
+	// }
+
+	@include breakpoint(small) {
+		font-size: calc(18 / 320 * 100vw);
```

Comment:
We can't do these vw multipliers for accessibility reasons. They prevent a user from increasing their font size. Will push up a change.